### ISSUE TITLE
[BUG] manual mesh leveling doesn't work 

### ISSUE THREAD

**Issue by mosheas** [cosmoderp/Marlin#10]
no_probe_menu firmware... When I go into the manual mesh leveling menu,I choose a point to calibrate the height of z, the numbers of height z have change, but physically there is no movement , what I should be do that it will work?

**Comment by techtosterone** [comment]
I agree! this is a legit bug. Assisted leveling menu works, Manual Mesh Leveling does not. Z babystepping doesn't work either
